phasespace/fromdecay/mass_functions.py

```diff
@@ -18,6 +19,8 @@ def mass_func(min_mass, max_mass, n_events):
             lambda lim: pdf.sample(1, limits=(lim[0], lim[1])), iterator
         )
 
+    mass_func.__name__ = "gauss"

```

Comment:
I agree, changing `__name__` is not a clean solution. Before, the name was not automatically set to "gauss", as the name of the function was "mass_func" by default, which is why I changed the name manually. However, I changed the function names now so that the `__name__` parameter is set to the correct value automatically (see commit 65e8ee2).